UserGuide.md

```diff
@@ -0,0 +1,89 @@
+<!-- markdownlint-disable -->
+<h1 align="center">
+  <br>
+    <img src="https://raw.githubusercontent.com/mauve/vscode-terraform/master/terraform.png" alt="logo" width="200">
+  <br>
+  vscode-terraform
+  <br>
+</h1>
+<h4 align="center">Adds intelligent auto-completion features for <a href="https://www.terraform.io/">Terraform</a> files</h4>
+
+
+<!-- markdownlint-enable -->
+<!-- markdownlint-disable MD002 MD013 MD041 -->
+
+## Features at a glance
+### Auto-completion features
+  1. `Terraform` high level types `(resource, data, variable, output,  module ...)` auto completion support.
+   - ![](https://github.com/zunlihu/Terraform-Language-Server/blob/master/images/Keyword.PNG) ```resource``` and `data` without bucket
+   - ![](https://github.com/zunlihu/Terraform-Language-Server/blob/master/images/variable.PNG) ```variable``` and `output` with bucket
+   - ![](https://github.com/zunlihu/Terraform-Language-Server/blob/master/images/modules.PNG) ```module``` with bucket
+  
+  2. `resource` types & `data` types auto completion support for [all providers](https://www.terraform.io/docs/providers/index.html), including azurerm, aws, google, alicloud and etc.
+  
+  3. `module` `source` parameter auto-completion support for [all providers](https://registry.terraform.io/) with required parameters auto-loading.
+  
+  4. `resource | data` default(required) parameters auto-completion.
+  
+  5. `resource | data | module | output | variable` property types auto completion support
+  
+### Others
+  1. Hyperlink for resource, data types and module sources, which is linked to Example Usage Web.
+
+## Install from *.vsix in Jenkins
```

Comment:
please remove the mentioning of Jenkins, I dont use it, feel free to mention Travis if you want to. However Travis doesn't really work with artefacts so I am planning to azure devops for building and testing